internal/pkg/template/templates/workloads/partials/cf/https-listener.yml

```diff
@@ -38,8 +38,13 @@ LoadBalancerDNSAlias{{$hostedZoneID}}:
           HostedZoneId: !GetAtt EnvControllerAction.InternalLoadBalancerHostedZone
           DNSName: !GetAtt EnvControllerAction.InternalLoadBalancerDNSName
           {{- else}}
+            {{- if $.CDNEnabled}}
```

Comment:
This change means that when the hosted zone is updated with the CloudFront resources, the stack Resource is not longer accurately named and described. I'd suggest we rename `LoadBalancerDNSAlias{{$hostedZoneID}}` to `PublicAccessDNSAlias{{$hostedZoneID}}` and the description accordingly to fit with the other changes like those to the Load Balancer Security Group.